tools/Replay.hpp

```diff
@@ -108,6 +154,68 @@ struct Stats {
     std::unordered_map<std::string, MessageStat> _recvd;
     std::unordered_map<std::string, MessageStat> _sent;
 
+    CPUStat GetProcessTimes() {
+
+        std::ifstream fileStat("/proc/" + std::to_string(getpid()) + "/stat");
```

Comment:
Hmm - this seems unhelpful. We measure higher resolution times in the SysStopwatch. We could split out the time there and get a much harder and hopefully more accurate number. Can you remove this lot and serialize the original tests's numbers from _timer.elapsedTime() or better.

Thanks.